pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -13,34 +13,36 @@ import (
 	"cloud.google.com/go/bigtable"
 	ot "github.com/opentracing/opentracing-go"
 	otlog "github.com/opentracing/opentracing-go/log"
+	"github.com/pkg/errors"
+	"golang.org/x/time/rate"
 
 	"github.com/cortexproject/cortex/pkg/chunk"
 	chunk_util "github.com/cortexproject/cortex/pkg/chunk/util"
 	"github.com/cortexproject/cortex/pkg/util"
 	"github.com/cortexproject/cortex/pkg/util/grpcclient"
-	"github.com/pkg/errors"
 )
 
 const (
-	columnFamily = "f"
-	columnPrefix = columnFamily + ":"
-	column       = "c"
-	separator    = "\000"
-	maxRowReads  = 100
+	columnFamily              = "f"
+	columnPrefix              = columnFamily + ":"
+	column                    = "c"
+	separator                 = "\000"
+	maxRowReads               = 100
+	bigtableMaxWriteBatchSize = 10000
```

Comment:
Why is hit hardcoded? How the value has been picked?

Reply:
Agreed, @tcl00taco are these values based on some constants related to bigtable. From what I understand the max batch size would be `100000` not `10000`

Reply:
@pracucci and @jtlisi Will remove hard coded value and make it configurable. The values were based on single cluster BigTable local test where I have not seen any query time out / delay. Thanks for the feedback and will make the changes.